core/src/trezor/wire/__init__.py

```diff
@@ -23,15 +23,13 @@
 
 """
 
-from micropython import const
 from typing import TYPE_CHECKING
 
-from storage.cache import InvalidSessionError
-from trezor import log, loop, protobuf, utils, workflow
-from trezor.enums import FailureType
-from trezor.messages import Failure
-from trezor.wire import codec_v1, context
-from trezor.wire.errors import ActionCancelled, DataError, Error, UnexpectedMessage
+from trezor import log, loop, protobuf, utils
+from trezor.wire import message_handler, protocol_common
+from trezor.wire.codec.codec_context import CodecContext
+from trezor.wire.context import UnexpectedMessageException
+from trezor.wire.message_handler import WIRE_BUFFER, failure
```

Comment:
while we're here, all `from trezor.wire` should be relative imports here.

(that btw makes it somewhat cleaner to see what is auto-imported together with `trezor.wire`)

Reply:
changed to relative imports here: https://github.com/trezor/trezor-firmware/pull/4357/commits/eda6521d11b2d045c1d77aaa58584f5e927d9271